### PR TITLE
added support of separate envs

### PR DESCRIPTION
**Pull request by vin-vlad**
Updated readme, added support of running project and browser-sync on different environments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,23 +5,36 @@ Install depedndencies
...
-SHOW_BROWSER - run in headless mode (`false` for headless mode)
+create separate ```.env``` files for different environments ```.env.dev .env.prod .env.stg``` etc. 
+.env 
+```
+HOST_URL
+SHOW_BROWSER
+WEBLOADER_URL
+PARTICIPANT_URL
+ENVIRONMENT_ID
+UID
```
Comment (mattstrom) [inline README.md:39]: Looks like you need to explicitly add the _.env*_ files to the commit. _.env_ is often ignored by _.gitignore_ by default (which is true for this repo).

Comment (mattstrom) [reply README.md:39]: As a possible alternative, in many of our other repos we use [node-config](https://www.npmjs.com/package/config) for configurations. I find it to be a bit more robust and flexible than `dotenv`.

Comment (vin-vlad) [reply README.md:39]: I didn't commit env files for security reasons. If you think it's ok to commit them, I'll add them to the repo

Comment (mattstrom) [reply README.md:39]: I see. That's a good practice then.

How do you plan to include the environment variables when the tests are run on CI/CD?

Comment (vin-vlad) [reply README.md:39]: I believe most of ci\cd tools provide env variables setup. 
Github Actions are also provide this functionality but I haven't had my hands on that. https://docs.github.com/en/actions/reference/environment-variables

Comment (mattstrom) [reply README.md:39]: This is where I think `node-config` is helpful. You can set up configuration files per environment that only include non-secret values that are checked in. And then using env vars set in Actions, you can inject secret values into the configuration.

Comment (mattstrom) [reply README.md:39]: The repo has an example of how we use node-config:
https://github.com/SentientTechnologies/integrations.evolv.ai/tree/master/config

Comment (mattstrom) [reply README.md:39]: For uniformity among our repos and for the extra functionality it brings, I think we should incorporate `node-config`.  Let me know if you need any guidance with this.

Comment (vin-vlad) [reply README.md:39]: I'll add `node-cofig` to this PR tomorrow morning with all configs

**Comment by vin-vlad** [comment]
I've replaced dotenv with node-config and created config files for environments

File: README.md (hunk 1 above)
Comment (mattstrom) [reply README.md:39]: This section is no longer relevant. You can reuse the language we use in other repos:
```markdown
To create a new environment configuration, copy the _config/default.js_ file into a new file in the same directory with the name _<environment>.js_.

> This repo leverages `node-config` for configuration files. See
> [here](https://github.com/lorenwest/node-config/wiki) for documentation about `node-config`.
> To make a local-only configuration, prefix the file name with _local-_.
> Any file matching `local-*` in the _config_ directory will be git-ignored, but make
> sure to not forcibly add the file to Git tracking.
```

**Review by mattstrom — APPROVED** [review]
There is one merge conflict, and I made a comment about updating a section in the README. But otherwise looks good. Thanks for converting your original work.

**Pull request merged**